tests/theia_automation_lsp/drivers/drv_config.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2019 Broadcom.
+# The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Broadcom, Inc. - initial API and implementation
+
+import os
+import sys
+import urllib.request as urllib2
+import urllib.parse as urlparse
+import zipfile
+import tarfile
+
+ENGINES_EXE = {
+    "chrome": "chromedriver",
+    "firefox": "geckodriver"
+}
+
+DRIVERS = {
+    "darwin": {
+        "firefox": "https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-macos.tar.gz",
```

Comment:
please make a list with the dependencies and versions use for the drivers in order to submit it for IP checks

Reply:
name: geckodriver
url: https://github.com/mozilla/geckodriver
license: Mozilla Public License 2.0
version: v0.24.0

name: chromedriver
url: https://chromedriver.chromium.org/
license: BSD license
version: 77.0.3865.40

Reply:
Some more details:
gecko:
Mozilla Public License 2.0
https://www.mozilla.org/en-US/MPL/2.0/
download from: https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-win64.zip


chrome:
https://chromedriver.chromium.org/
BSD license
https://cs.chromium.org/chromium/src/LICENSE
download from: https://chromedriver.storage.googleapis.com/77.0.3865.40/chromedriver_win32.zip

Reply:
Cannot update/create cq as I'm not committer AFAIU

Reply:
I will create the request

Reply:
status: waiting for approval